modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -161,16 +163,25 @@
 			<return type="void" />
 			<description>
 				Like [method @GlobalScope.print], but prints only when used in debug mode.
```

Comment:
This line might not be correct. I'm not sure if the implementation is supposed to treat `print_debug` as a no-op without the debugger on, or if this is supposed to state that `print_debug` only gives debug info with the debugger on.

Reply:
Seems it was just a documentation error: #24804
I'll update PR to fix this later.

Reply:
Fixed